doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,35 @@
-=======================
-Release Build Creation
-=======================
+=========================================================
+Compile & Build Complete Cortx Stack Via Docker Container
+=========================================================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
```

Comment:
This is the prerequisites not the specifications. My suggestion to rename it as:

# Prerequisites

Reply:
thanks, done